modules/move-etcd-different-disk.adoc

```diff
@@ -37,116 +77,7 @@ $ oc debug node/<node_name>
 +
 Note the device name of the new disk reported by the `lsblk` command.
 
```

Comment:
We're missing the step that I suggested in my last review here. :slightly_smiling_face: 

```adoc
. Encode the following script in base64, noting the output:
+
[source,bash]
----
#!/bin/bash
set -uo pipefail

for device in <device_type_glob>; do # <1>
----
<1> Replace `<device_type_glob>` with a shell glob for your block device type. For SCSI or SATA drives, use `/dev/sd*`; for virtual drives, use `/dev/vd*`; for NVMe drives, use `/dev/nvme*[0-9]\*n*`.
```

Note that I've cut the contents of the script short just for readability in a comment. :slightly_smiling_face: